sovtokenfees/doc/Interface/README.md

```diff
@@ -377,3 +377,81 @@ Example:
     }
 }
 ```
+
+## How to set fees for an action
+ For setting fees for an action we need to make the following steps:
+ * Send a SET_FEES txn with appropriate amount for required alias. 
+ For example, we have an alias, like "add_new_steward" (and we want to set fees for adding new nym action). For setting fees for this alias, we need to send a SET_FEES transaction with map {"add_new_steward": 42}.
+ Txn will be looked as:
+ ```
+{
+    "reqId": <int>,             //random identifier
+    "protocolVersion": <int>,   // the version of the client/node communication protocol
+    "operation": {
+        "type": "20000",
+        "fees": {
+            "add_new_steward": 42,,
+        }
+    },
+}
+```
+ * After that, we need to add metadata into default auth constraint for action "add new nym".
+ For this example, txn for changing metadata for default auth_rule will be looked as:
+ ```
+{
+    'operation': {
+           'type':'120',
+           'constraint':{
+                    'constraint_id': 'ROLE', 
+                    'role': '0',
+                    'sig_count': 1, 
+                    'need_to_be_owner': False, 
+                    'metadata': {'fees': 'add_new_steward'}
+           }, 
+           'field' :'role',
+           'auth_type': '1', 
+           'auth_action': 'ADD',
+           'new_value': '2'
+    },
+    
+    'identifier': <str: some identifier>,
+    'reqId': <int: timestamp>,
+    'protocolVersion': 1,
+    'signature': <str: signature>
+}
+```
+
+The pool performs the following validation for the given example:
+* doDynamicValidation for "adding new steward" nym (_`from indy-node's side`_);
+    * making general role's authorization (_`from indy-node's side`_)
+    * making fees specific validation, using metadata field (_`from plugin's side`_)
+        * lookup through config state for getting amount of "add_new_steward" alias (_`from plugin's side`_)
+        * making can_pay_fees validation (_`from plugin's side`_)
+        
+### Notes:
+* The order of previous steps is very important. First of all SET_FEES is required, then - AUTH_RULE.
+* SET_FEES is "updating" transaction, so that it appends new aliases to the existing FEEs map (either adding or overriding aliases). For example, if current fees are {A: 1, B: 2} then after sending SET_FEES transaction with {A: 42, C:3}, the resulted map will look like {A: 42, B: 2, C:3}. 
+* Setting fees without adding or changing metadata in corresponded auth_rule does not make sense. 
```

Comment:
Better to say:
`Setting fees without adding or changing metadata in corresponding auth_rule doesn't have any effect. `